### ISSUE TITLE
Topbot refactor fails on opening a single IDF

### ISSUE THREAD

**Issue by Manangka** [Deltares/imod-python#82]
In GitLab by @Huite on Jul 29, 2019, 14:19

The second last line fails with an IndexError if only one file is loaded, which has a top and bottom.
 

```python   
    if all_have_z:
        if haslayer:
            top = np.array(tops)[unique_indices]
            bot = np.array(bots)[unique_indices]
            dz = top - bot
            z = top - 0.5 * dz
            if top[0] < top[1]:  # decreasing coordinate
                dz *= -1.0
```

Size == 1 should be special cased.

**Comment by Manangka** [comment]
In GitLab by @Huite on Jul 30, 2019, 09:27

closed via commit abf8c37fdcac5003a809332a22e01b394965139c

**Comment by Manangka** [comment]
In GitLab by @Huite on Aug 6, 2019, 09:32

mentioned in commit abf8c37fdcac5003a809332a22e01b394965139c
